lib/common.sh

```diff
@@ -90,20 +90,52 @@ export SSH_PUB_KEY_CONTENT
 FILESYSTEM=${FILESYSTEM:="/"}
 
 # Environment variables
-# M3PATH : Path to clone the Metal3 Development Environment repository
```

Comment:
If we need to keep these all in https://github.com/metal3-io/metal3-dev-env/blob/main/vars.md, then we need to update it (CAPI, IPAM repo/branch notes + with commit vars for each)

Reply:
I don't understand what you are saying  could you please clarify?

Reply:
Sorry, maybe I highlighted not the correct line. Request was: if you check the file, there we provide - CAPM3REPO/CAPM3BRANCH or BMOREPO/BMOBRANCH, but not IPAM or CAPI. Also, var `REPO`COMMIT is completely missing. So my suggestion is: either remove existing ones as well OR add missing ones to the list.